java/core/src/test/org/apache/orc/impl/TestWriterImpl.java

```diff
@@ -29,7 +29,7 @@
 import org.apache.orc.Reader;
 import org.apache.orc.TypeDescription;
 import org.apache.orc.Writer;
-import org.apache.orc.*;
+import org.apache.orc.StripeInformation;
```

Comment:
It looks like the maven-checkstyle-plugin doesn't check the test code, but it's best to sort the imports in the group alphabetically

Reply:
@guiyanakuang Thanks for the feedback. Shall be aware of this in future. I was hesitant in using Intellij Optimize imports, as it was changing the static import order too.
Updated the code to reflect the order.

Reply:
You can try using the plugin https://plugins.jetbrains.com/plugin/1065-checkstyle-idea so that you can use checkstyle.xml from your ORC project instead of IDEA's default code checking.

<img width="919" alt="image" src="https://github.com/apache/orc/assets/4069905/7375ac17-b446-4d25-87dd-5b55a182b91c">

I've been using it for a while and it works great.

https://github.com/apache/orc/blob/1c83941a213a55c1d465bae1c02e49e260f7b9cd/java/checkstyle.xml#L42